CONTRIBUTING.md

```diff
@@ -89,6 +89,7 @@ MONAI tests are located under `tests/`.
 
 - The unit test's file name follows `test_[module_name].py`.
```

Comment:
Should we change this to "For unit tests that use single GPU, file name follows test_[module_name].py"